lms/templates/ccx/student_admin.html

```diff
@@ -2,10 +2,19 @@
 
 <section>
   <h2>${_('Student Grades')}</h2>
-  <p>
-    <a href="${gradebook_url}">${_('View gradebook')}</a>
-  </p>
-  <p>
+  <p class="ccx-grade-book-download">
+    <i class="fa fa-download" aria-hidden="true"></i>
     <a href="${grades_csv_url}">${_('Download student grades')}</a>
   </p>
+  <br/>
+  <div>
+    <iframe src='${gradebook_url}' id="ccx_grade_book_iframe"
+            name="ccx_grade_book_iframe"
+            frameBorder="0"
+            scrolling="no"
+            height="360px"
+            class="ccx-grade-book">
+      <p>Iframe not supported</p>
+    </iframe>
```

Comment:
@sarina gradebook_url only loads the first page of the gradebook. 

This PR has actually been waiting for the pagination PR #10250 to merge so we avoid loading the entire gradebook when the tab is opened. Now we're just loading the first page of it.